packages/app/obojobo-repository/server/models/draft_summary.js

```diff
@@ -90,6 +97,49 @@ class DraftSummary {
 			})
 	}
 
+	static fetchAllDraftRevisions(draftId) {
+		const query = `
+			SELECT
+				drafts_content.id,
+				drafts_content.draft_id,
+				drafts_content.created_at,
+				drafts_content.content,
+				repository_map_user_to_draft.user_id,
+				users.first_name,
+				users.last_name
+			FROM drafts_content
+			JOIN repository_map_user_to_draft
+				ON repository_map_user_to_draft.draft_id=drafts_content.draft_id
+			JOIN users
+				ON repository_map_user_to_draft.user_id=users.id
+			WHERE drafts_content.draft_id=$[draftId]
+			ORDER BY drafts_content.created_at DESC;
```

Comment:
Summoning @iturgeon

Reply:
Yea, this is an awful lot of data and unbound by any restrictions.  While it may work for now, we need this to be paginated or not return `drafts_content.content`.